### ISSUE TITLE
Missing Polymer 2 support

### ISSUE THREAD

**Issue by philippw** [PolymerElements/platinum-sw#118]
### Description
When trying to install `polymer-sw` I am asked to download Polymer 1.0.

According to the official [What's new in 2.0](https://www.polymer-project.org/2.0/docs/about_20) page `platinum-sw` ought to be compatible with 2.0.

### Expected outcome

It should just work with Polymer 2.0.

### Actual outcome

We're downloading Polymer 1.9.2.

### Steps to reproduce

1. Install `polymer-cli` via NPM
2. Run `polymer init`
3. Run the offical install instructions of `platinum-sw`: `bower install --save PolymerElements/platinum-sw` 
4. See that you're being asked to also install Polymer 1.9.2
```
PS C:\Users\Administrator\Documents\Polymer\Lab 11 - Offline Apps> bower install --save PolymerElements/platinum-sw
bower platinum-sw#*             cached https://github.com/PolymerElements/platinum-sw.git#1.3.0
bower platinum-sw#*           validate 1.3.0 against https://github.com/PolymerElements/platinum-sw.git#*
bower polymer#^1.0.0            cached https://github.com/Polymer/polymer.git#1.9.2
bower polymer#^1.0.0          validate 1.9.2 against https://github.com/Polymer/polymer.git#^1.0.0
bower sw-toolbox#^3.1.1         cached https://github.com/GoogleChrome/sw-toolbox.git#3.6.1
bower sw-toolbox#^3.1.1       validate 3.6.1 against https://github.com/GoogleChrome/sw-toolbox.git#^3.1.1
bower webcomponentsjs#^0.7.24   cached https://github.com/Polymer/webcomponentsjs.git#0.7.24
bower webcomponentsjs#^0.7.24 validate 0.7.24 against https://github.com/Polymer/webcomponentsjs.git#^0.7.24
bower webcomponentsjs#^v1.0.0   cached https://github.com/Polymer/webcomponentsjs.git#1.0.1
bower webcomponentsjs#^v1.0.0 validate 1.0.1 against https://github.com/Polymer/webcomponentsjs.git#^v1.0.0

Unable to find a suitable version for polymer, please choose one by typing one of the numbers below:
    1) polymer#^1.0.0 which resolved to 1.9.2 and is required by platinum-sw#1.3.0
    2) polymer#^2.0.0 which resolved to 2.0.1 and is required by offline-app-demo
```

### Browsers Affected
- [X] Chrome
- [X] Firefox
- [X] Safari 9
- [X] Safari 8
- [X] Safari 7
- [X] Edge
- [X] IE 11
- [X] IE 10

**Comment by DominicWrege** [comment]
It would be very nice to have it in polymer 2

**Comment by e111077** [comment]
Fixed by #112

**Comment by e111077** [comment]
Though the team and creator of this element highly recommend that you try to move off of this element and use [workbox](https://github.com/GoogleChrome/workbox) or the older [sw-toolbox](https://github.com/GoogleChrome/sw-toolbox)
